### ISSUE TITLE
Buff the Magical Energy converter slightly or make more tiers (its more useless than solar as is and gated behind TC)

### ISSUE THREAD

**Issue by Alastors** [GTNewHorizons/GT-New-Horizons-Modpack#7540]
#### Which modpack version are you using?
2.1.0.3a
#
#### If in multiplayer; On which server does this happen? 
Mine
#
#### What do you suggest instead/what changes do you propose?

I propose that the Magical energy converter's get their costs reduced. I propose the converter and absorber get reduced from the two tiers higher than they are useful for to just 1 tier higher. They are a pain and a half to make and use and are not worth the effort as is. But by reducing the lv one to mv costs and the hv one to ev costs it would make a very painful effort but actually slightly worth it for those who are willing to go into thaumcraft. Like to be realistic the beacon should probably be removed from the recipe but its fine as is. 

But, for what they do the cost is much too high. For one to mass produce the nodes you need for it you really need to get into magic bees and the combinator or another aspect moving machine to move the aspects into another node and then sit around for 6 hours to just make a single node thats good enough to power the highest tier. Plus the fact that the bees required for that method are an absolute nightmare to get to and the combinator takes stuff that is typically done later in the game because of the way thaumcraft is usually gated.

So I propose you keep the recipes the same with the same costs except the stainless steel in the lv one is reduced to aluminium (still with the beacon so) and the circuits are reduced to mv, the mv ones require stainless and hv circuits (cleanroom circuits maybe?), and so forth and so on?

tl;dr: Drop the gates on the magical energy converters by one tier so that they become at least slightly usable they arent even close to worth the trouble as they currently are. They are a lot harder to make than solar and as of right now a lot less worth it

(shouldnt they be at least a just a lil easier to make than solar since they are gated behind thaumcraft mechanics? I mean gating thaumcraft like its mundane equivalent makes thaumcraft not even worth the effort for so many people you know?)

**Comment by Alastors** [comment]
My point is they cost a lot more than solar as of right now and are basically gated the exact same way as solar so they are desperate for a buff

**Comment by Alastors** [comment]
![Uploading Screen Shot 2021-02-27 at 12.05.50 AM.png…]()

**Comment by Alastors** [comment]
<img width="1440" alt="Screen Shot 2021-02-27 at 12 05 57 AM" src="https://user-images.githubusercontent.com/78517796/109377323-31bc9b80-7890-11eb-816e-04ffc9262a44.png">
